.travis.yml

```diff
@@ -13,74 +13,13 @@ install:
   - pip install --upgrade --pre tox
 env:
   matrix:
-    - TOXENV=py27
```

Comment:
this commit have me a little shock until i realized its  a temporary measure - i will put WIP in the pr title until those details are resolved

Reply:
Removed the commit.
https://travis-ci.org/pytest-dev/pytest/builds/490709302?utm_source=github_status&utm_medium=notification shows that it works.

But pip 19.0.2 is likely to fix it also anyway.